app/views/admins/_stats.haml

```diff
@@ -0,0 +1,50 @@
+- content_for :head do
+  = stylesheet_link_tag :admin
+
+%h1= t(".usage_statistic")
+
+.row
+  = form_tag("/admins/stats", method: "get", class: "col-sm-12 form-inline segment-selection") do
+    .col-sm-2.col-sm-push-8
+      %select.form-control{name: "range"}
+        - ["daily", "week", "2weeks", "month"].each do |range|
```

Comment:
Sorry, I thought % was the same than for the HTML so I removed the `-`. Now it works.